sysobject.ids

```diff
@@ -3202,6 +3204,7 @@
 171.10.75.15	D-Link	NETWORKING	DES-1210-28/ME
 171.10.76.4	D-Link	NETWORKING	DGS-1216T
 171.10.76.5	D-Link	NETWORKING	DGS-1224T
+171.10.76.32.1	D-Link	NETWORKING	UNKNOWN
```

Comment:
As above, why set model to `UNKNOWN` here ?

Reply:
Same question as above: do you know if many models share that sysObjectID values ?

Reply:
`So can you answer that question: do you know if many models share that sysObjectID values ?`
Yes I can answer, no I don't if many models share the same values.

Reply:
Here an entry with `UNKNOWN` is useless as we still have the partial match on `171`.
```suggestion
```
Anyway if you manage to find what model should be set, just replace `UNKNOWN` by the expected model.